lib/rules/indent.js

```diff
@@ -922,7 +922,15 @@ module.exports = {
 
             parameterParens.add(openingParen);
             parameterParens.add(closingParen);
-            offsets.setDesiredOffset(openingParen, sourceCode.getTokenBefore(openingParen), 0);
+
+            let offsetFrom = sourceCode.getTokenBefore(openingParen);
+
+            // If the node's callee is a TaggedTemplateExpression, use callee's tag as a offset from.
```

Comment:
I think we can get rid of this comment - the code actually reads a lot like English in this case 😊